pages/juang961225/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>carrusel</title>
+    <link rel="stylesheet" href="../juang961225/style.css">
+</head>
+<body>
+<!--contenedor general-->
+    <main class="main">
+<!--titulo principal-->
+        <h2 class="main__title">word of warcraft class</h2>
+<!--contenedor de las cartas-->
+        <div class="main__container">
+<!--boton izquierdo de las cards-->
+        <button class="main__button">
+            <h2 class="main__container-button"><</h2>
```

Comment:
I think you may change the tag H2 here because it is not pretty common or a good practice to use headline elements as the value for a button.

Reply:
Also, If this button is an arrow, you could use an image and a better name like `main__arrow`.

Reply:
resolved